extensions/omni_httpd/docs/headers.md

```diff
@@ -8,4 +8,73 @@ select omni_http.http_header_get(request.headers, 'host') as host;
 select omni_http.http_header_get_all(request.headers, 'accept') as accept;
 ```
 
-The header name these functions take is case-insensitive.
\ No newline at end of file
+The header name these functions take is case-insensitive.
+
+## Content-Length
+
+`omni_httpd` automatically sets the `Content-Length` header for any non-null response body.
+
+For instance, having:
+
+```sql
+create extension omni_httpd cascade;
+
+set omni_httpd.init_port to 8000;
```

Comment:
This is ineffectual because only the extension installation script uses these parameters.

This can lead to confusion. Perhaps for documenting purpose, we can remove this (instead of moving it above `create extension`)

Reply:
Removed. Now using the default `8080` port for the curl examples instead.